pkgs/by-name/in/invoiceplane/package.nix

```diff
@@ -1,32 +1,103 @@
-{ lib, stdenv, fetchurl, unzip, nixosTests }:
+{
+  lib,
+  fetchFromGitHub,
+  nixosTests,
+  fetchYarnDeps,
+  nodejs,
+  php,
+  yarnConfigHook,
+  yarnBuildHook,
+  yarnInstallHook,
+  nodePackages,
+  python3,
+  pkg-config,
+  libsass,
+  stdenv,
+}:
 
-stdenv.mkDerivation rec {
+php.buildComposerProject (finalAttrs: {
   pname = "invoiceplane";
   version = "1.6.1";
 
-  src = fetchurl {
-    url = "https://github.com/InvoicePlane/InvoicePlane/releases/download/v${version}/v${version}.zip";
-    hash = "sha256-66vXxE4pTUMkmPalLgJrCt2pl2BSWOJ3tiJ5K5wspYY=";
+  src = fetchFromGitHub {
+    owner = "InvoicePlane";
+    repo = "InvoicePlane";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-rVt3HkAh3zBVwt1a4ZGiuC/Khvb3Ugk42hT5mul4qRA=";
   };
 
-  nativeBuildInputs = [ unzip ];
+  vendorHash = "sha256-qYjNn7VQGkUfJ74coUpUOfAOkgi3eLoo/ITpcOcTenk=";
 
-  sourceRoot = ".";
+  buildInputs = [ libsass ];
 
-  installPhase = ''
-    mkdir -p $out/
-    cp -r ip/. $out/
+  nativeBuildInputs = [
+    yarnConfigHook
+    yarnBuildHook
+    yarnInstallHook
+    # Needed for executing package.json scripts
+    nodePackages.grunt-cli
+    pkg-config
+    (python3.withPackages (ps: with ps; [ distutils ]))
+    stdenv.cc
```

Comment:
Isn't this the default already?

Reply:
probably not in `php.buildComposerProject

Reply:
without that build input i get `make: c++: No such file or directory`